server/core/tablecodec.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package core
+
+import (
+	"bytes"
+	"encoding/binary"
+	"github.com/juju/errors"
+)
+
```

Comment:
I guess we can add some references to the table codec implementation on tidb.

Reply:
@siddontang any comments on this?